.github/workflows/images.yaml

```diff
@@ -25,8 +25,12 @@ jobs:
     strategy:
       matrix:
         include:
-          - name: cilium-cli
+          - name: cilium-cli-ci
             dockerfile: ./Dockerfile
+            platforms: linux/amd64
+          - name: cilium-cli
+            dockerfile: ./Dockerfile-cli-only
```

Comment:
Looks like this should now be `./Dockerfile`

Reply:
Fixed in last commit, had to rebase from main as well